pkg/client/rules.go

```diff
@@ -32,18 +47,31 @@ func (r *CortexClient) CreateRuleGroup(ctx context.Context, namespace string, rg
 
 // DeleteRuleGroup creates a new rule group
 func (r *CortexClient) DeleteRuleGroup(ctx context.Context, namespace, groupName string) error {
+	path := rulerAPIPath
+	if r.legacy {
+		path = legacyAPIPath
+	}
 	escapedNamespace := url.PathEscape(namespace)
 	escapedGroupName := url.PathEscape(groupName)
+	path += "/" + escapedNamespace + "/" + escapedGroupName
 
-	_, err := r.doRequest("/api/prom/rules/"+escapedNamespace+"/"+escapedGroupName, "DELETE", nil)
+	log.WithFields(log.Fields{
+		"url": path,
+	}).Debugln("path built to request rule group")
```

Comment:
**nitpick:** I think the `doRequest` function logs the path of the request. If so, this log line should be removed.

Reply:
I agree. I'll remove it.